apps/web/composables/useProducts/useProducts.ts

```diff
@@ -16,13 +17,13 @@ export const useProducts: UseProductsReturn = () => {
   /**
    * @description Function for fetching products.
    * @example
-   * getProducts();
+   * getFacet(@props: FacetSearchCriteria)
    */
-  const fetchProducts: FetchProducts = async () => {
+  const fetchProducts: FetchProducts = async (slug: FacetSearchCriteria) => {
```

Comment:
I would rename slug with `params`, because I think a slug is more like an url of some sort